pilot/adapter/config/file/config.go

```diff
@@ -0,0 +1,116 @@
+// Copyright 2017 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package file
+
+import (
```

Comment:
run goimports

Reply:
I ran this, which printed out files ... not sure it changed anything though:

```
goimports pilot/adapter/config/file/*
```

Reply:
Use the `-w` flag to make goimports write results to the input file rather than stdout.

Reply:
Ah thanks.  I missed that when I ran `goimports -help`.

Done.